### ISSUE TITLE
Setting examples for JFrog support missing / JForg not supported

### ISSUE THREAD

**Issue by BAAZKonqi** [aliasadidev/vscode-npm-gui#55]
I'm trying to use this extension with a JFrog artifactory located on a company server. It is not working. Now, I am not sure if it is due to lacking support or to a misconfiguration on my end. The README states, that JFrog is supported, however in the example settings.json there is no example for JFrog.

**Comment by aliasadidev** [comment]
You're right, in the past I've only checked the query endpoint and it was correct bud didn't test the version endpoint. Unfortunately, there is no endpoint for getting the package version in JFrog.
